### PR TITLE
 Raiders of the Lost Docs

### PR DESCRIPTION
**Pull request by mjmagee991**
Adds high-level documentation for the current state of the BMS

### CODE REVIEW

**Comment by mjmagee991** [comment]
[BMS Hand-Off Documentation](https://trello.com/c/B5n7AZCP/51-bms-hand-off-documentation)

File: include/BMS/dev/BQ76952.hpp [hunk 1]
```diff
@@ -163,15 +165,63 @@ class BQ76952 {
...
+        3,
+        4,
+        5,
+        6,
+        8,
+        10,
+        12,
+        14,
+        15,
+    };
```
Comment (AarC10) [inline include/BMS/dev/BQ76952.hpp:223]: Maybe make this more consistent with each other

**Review by AarC10 — CHANGES_REQUESTED** [review]

File: docs/source/index.rst [hunk 2]
```diff
@@ -28,12 +28,20 @@ For a more detailed description of the requirements, refer to the
 The DEV1-BMS project is in its rampup phase. Currently documentation is being
 generated for the project including requirements and implementation design.
 
+The DEV1-BMS project is currently in the hardware bringup phase. Software
```
Comment (DeLozM8) [inline docs/source/index.rst:31]:
```suggestion
The DEV1-BMS project is currently in the hardware bring up phase. Software
```

File: include/BMS/BMS.hpp [hunk 3]
```diff
@@ -80,7 +80,7 @@ class BMS {
      * Have to know the size of the object dictionary for initialization
      * process.
      */
-    static constexpr uint16_t OBJECT_DIRECTIONARY_SIZE = 30;
+    static constexpr uint16_t OBJECT_DIRECTIONARY_SIZE = 42;
```
Comment (DeLozM8) [inline include/BMS/BMS.hpp:83]:
```suggestion
    static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 42;
```

File: src/dev/BQ76952.cpp [hunk 4]
```diff
@@ -15,12 +15,124 @@
...
+}
+
+/**
+ * Read in the balance state of the given cell. This will communicate with the
+ * BQ to determine the state.
+ *
+ * @param[in] obj The CANopen stack object dictionary, used to determine
+ *                the cell number.
+ * @param[in] node The CANopen stack node (not used)
+ * @param[out] buf THe buffer to populate with data
```
Comment (DeLozM8) [inline src/dev/BQ76952.cpp:51]:
```suggestion
 * @param[out] buf The buffer to populate with data
```

File: targets/BMS/main.cpp [hunk 5]
```diff
@@ -110,13 +110,13 @@ int main() {
...
+    EVT::core::IO::I2C& i2c = EVT::core::IO::getI2C<IO::Pin::PB_6, IO::Pin::PB_7>();
 
     // Initialize the timer
     DEV::Timerf302x8 timer(TIM2, 100);
 
     // Initialize the EEPROM
-    EVT::core::DEV::M24C32 eeprom(0x50, i2c);
+    EVT::core::DEV::M24C32 eeprom(0x53, i2c);
 
     // Intialize the logger
```
Comment (DeLozM8) [inline targets/BMS/main.cpp:121]:
```suggestion
    // Initialize the logger
```

File: targets/BMS/main.cpp [hunk 6]
```diff
@@ -127,15 +127,18 @@ int main() {
     BMS::BQSettingsStorage bqSettingsStorage(eeprom, bq);
 
     // Intialize the Interlock
```
Comment (DeLozM8) [inline targets/BMS/main.cpp:129]:
```suggestion
    // Initialize the Interlock
```

File: targets/BMS/main.cpp [hunk 7]
```diff
@@ -127,15 +127,18 @@ int main() {
...
-    // TODO: Determine actual interlock GPIO
-    IO::GPIO& interlockGPIO = IO::getGPIO<IO::Pin::PB_0>(IO::GPIO::Direction::INPUT);
+    IO::GPIO& interlockGPIO = IO::getGPIO<IO::Pin::PA_3>(IO::GPIO::Direction::INPUT);
     BMS::DEV::Interlock interlock(interlockGPIO);
 
     // Intialize the alarm pin
```
Comment (DeLozM8) [inline targets/BMS/main.cpp:133]:
```suggestion
    // Initialize the alarm pin
```

**Review by DeLozM8 — CHANGES_REQUESTED** [review]

**Review by AarC10 — APPROVED** [review]

**Review by DeLozM8 — APPROVED** [review]

**Comment by cbolles** [comment]
If nothing else, I can say I left lasting impact in terms of shear number of typos.

**Pull request merged**